Assets/AirConsole/scripts/editor/Extentions.cs

```diff
@@ -131,16 +131,14 @@ class Extentions {
 		}
 
 		public static string GetLocalAddress () {
-
 			string localIP = "";
-  
-			foreach (IPAddress ip in Dns.GetHostEntry(Dns.GetHostName()).AddressList) {
-                
-				if (ip.AddressFamily == AddressFamily.InterNetwork) {
-					localIP = ip.ToString ();
-					break;
-				}
-			}
+
+			using (Socket socket = new Socket(AddressFamily.InterNetwork, SocketType.Dgram, 0)) {
```

Comment:
screwed up indentation.
Plus what happens if 8.8.8.8 stops supporting this request?

Reply:
It doesn't actually connect to 8.8.8.8. But I updated the IP to be a LAN IP instead which seems to do the job correctly